selfdrive/car/gm/interface.py

```diff
@@ -165,61 +167,20 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
       ret.lateralTuning.pid.kf = 1.  # get_steer_feedforward_volt()
       ret.steerActuatorDelay = 0.2
 
-    elif candidate == CAR.MALIBU:
-      ret.mass = 1496.
-      ret.wheelbase = 2.83
-      ret.steerRatio = 15.8
-      ret.centerToFront = ret.wheelbase * 0.4  # wild guess
-
-    elif candidate == CAR.HOLDEN_ASTRA:
-      ret.mass = 1363.
-      ret.wheelbase = 2.662
-      # Remaining parameters copied from Volt for now
-      ret.centerToFront = ret.wheelbase * 0.4
-      ret.steerRatio = 15.7
-
     elif candidate == CAR.ACADIA:
       ret.minEnableSpeed = -1.  # engage speed is decided by pcm
-      ret.mass = 4353. * CV.LB_TO_KG
-      ret.wheelbase = 2.86
-      ret.steerRatio = 14.4  # end to end is 13.46
-      ret.centerToFront = ret.wheelbase * 0.4
       ret.steerActuatorDelay = 0.2
       CarInterfaceBase.configure_torque_tune(candidate, ret.lateralTuning)
```

Comment:
most are torque now, want to make another PR to put this on top?

Reply:
I'm not sure if you're referring to putting another field into the CarSpecs or just making it the default for GM. If you mean the latter, is there a way to unset the torque tune for the couple of cars that don't use it?

Reply:
make it the default for GM. you can reinit the pid tune with
```ret.lateralTuning.init('pid')```